spring-ai/src/main/java/com/baeldung/imagegen/Application.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.imagegen;
+
+import org.springframework.ai.autoconfigure.vectorstore.mongo.MongoDBAtlasVectorStoreAutoConfiguration;
+import org.springframework.ai.autoconfigure.vectorstore.redis.RedisVectorStoreAutoConfiguration;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.PropertySource;
+
+@SpringBootApplication(exclude = {
+    RedisVectorStoreAutoConfiguration.class,
+    MongoDBAtlasVectorStoreAutoConfiguration.class
```

Comment:
Can we add a comment explaining why these are excluded?

Reply:
review incorporated.